### PR TITLE
added placement model

### LINKED ISSUE #3: Create Placement model
You have to create the model for Placements here. This model is supposed to store the data for all the placements that are updated by SPO. The fields here are really straightforward -

- Placement Name
- Company
- Role
- Description
- Eligible batches
- Eligible branches
- Eligible Programmes
- Deadline

### PR DESCRIPTION
**Pull request by Shubham63-hub**
Closes #3

### CODE REVIEW

**Comment by Shubham63-hub** [comment]
updated placement model, changed default of jsonfields to list from dict.

**Comment by shashank1310** [comment]
description ka max length  1500 kar de

**Comment by Shubham63-hub** [comment]
done! increased the length of description and also changed timezone to  Asia/Calcutta

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+db.sqlite3
```
Comment (SomTambe) [inline .gitignore]: Add a newline at the end of the file.

File: placement/models.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+class Placement(models.Model):
+    placement_text = models.CharField(max_length=300)
+    company = models.CharField(max_length=300)
+    role = models.CharField(max_length=100)
+    description = models.CharField(max_length=1500)
+    eligible_batches = models.JSONField(default= list )
```
Comment (SomTambe) [inline placement/models.py]: Change `eligible_batches = models.JSONField(default= list )` to `eligible_batches = models.JSONField(default=list)`. Notice the spaces and make the same changes for all the files.

File: placement/models.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+class Placement(models.Model):
+    placement_text = models.CharField(max_length=300)
```
Comment (SomTambe) [inline placement/models.py]: change the name from `placement_text` to `placement_name` or something similar.

**Review by SomTambe — CHANGES_REQUESTED** [review]
Make the requested changes.

File: placement/models.py (hunk 3 above)
Comment (Shubham63-hub) [reply placement/models.py]: changed placement_text to placement_name

File: placement/models.py (hunk 2 above)
Comment (Shubham63-hub) [reply placement/models.py]: corrected the spaces

File: .gitignore (hunk 1 above)
Comment (Shubham63-hub) [reply .gitignore]: added a new line.

**Pull request closed without merging**